pos_sub_topic: "hedge_pos_ang"
wheel_covariance_topic: "wheel_odom_with_covariance"
pose_pub_topic: "parsed_pose"
frame_id: "map"
child_frame: "beacon_frame"
pose_covariance_diagonal: [0.01, 0.01, 0, 0, 0, 0]
min_linear_speed_covariance: 0.01
max_linear_speed_covariance: 0.001